config/sevntu_suppressions.xml

```diff
@@ -5,4 +5,8 @@
         "https://checkstyle.org/dtds/suppressions_1_1.dtd">
 
 <suppressions>
+	<!-- Messages.java are generated by the Eclipse localization wizard, therefore field order is generally not managed by human editors -->
+    <suppress checks="CustomDeclarationOrder"
```

Comment:
Indentation to fix

Reply:
I guess you are talking about the tab that slipped in at the comment? If so, we should have a checkstyle config for the configuration files. I have that at work to avoid things like duplicate ids, repeated checks etc.

Reply:
>we should have a checkstyle config for the configuration files

We usually do. But you disable bunch of checks.
So in this repo we prone to such minors.

Reply:
It is also more than 100 characters.